pymc3/model.py

```diff
@@ -1000,6 +1001,27 @@ def flatten(self, vars=None, order=None, inputvar=None):
         flat_view = FlatView(inputvar, replacements, view)
         return flat_view
 
+    def check_test_point(self, test_point=None, round_vals=2):
+        """Checks log probability of test_point for all random variables in the model.
+
+        Parameters
+        ----------
+        test_point : Point
+            Point to be evaluated.
+            if None, then all model.test_point is used
+        round_vals : int
+            Number of decimals to round log-probabilities
+
+        Returns
+        -------
+        Pandas Series
+        """
+        if test_point is None:
+            test_point = self.test_point
+
+        return Series({RV.name:np.round(RV.logp(self.test_point), round_vals) for RV in self.basic_RVs}, 
```

Comment:
Any reason to round?  

Also, what do you think about returning a `pd.DataFrame` with columns `log-probability` and `test_point`?

Reply:
I did not think that displaying `logp`s to high precision was helpful, particularly since we are mostly looking for `nan`s. 

The DF is a good idea! Let me try and do that.

Reply:
I guess the problem with this is that the `test_point` for some variables is vector-valued, so it would not comport well to being displayed in a DataFrame cell.